### DISCUSSION TITLE
Move from Travis-ci to another CI system

### DISCUSSION THREAD

**Discussion by briancolfer** [vcr/vcr#855] [General]
It is very unnerving that the test badge shows it is failing ... and that travis-ci says this no longer an active repository.

I would like to migrate the tests to another continual integration system such as github actions.

The alternatives are to fix the Travis-ci implementation, use circle-ci, shippable, gitlab ... The most direct route is github actions

**Comment by krainboltgreene** [comment]
I believe we've talked about this in past PRs and the team agrees with you. If you want to spearhead that it would be fantastic.

**Comment by briancolfer** [comment]
Yes I am happy to do that ... I'll add something this weekend.

**Comment by olleolleolle** [comment]
(I hit restart build on the failed job, perhaps it was a flakey test.)

**Comment by mcfiredrill** [comment]
I've moved a few repos to from travis to github actions recently and I am pretty happy with it.
